pkg/apis/resource/types.go

```diff
@@ -186,6 +237,28 @@ const ResourceSliceMaxSharedCapacity = 128
 const ResourceSliceMaxDevices = 128
 const PoolNameMaxLength = validation.DNS1123SubdomainMaxLength // Same as for a single node name.
 
+// Defines the max number of SharecCounters that can be specified
+// in a ResourceSlice. This is used to validate the fields:
+// * spec.sharecCounters
+const ResourceSliceMaxSharecCounters = 32
+
+// Defines the max number of SharedCounters from which a device
```

Comment:
comment needs to be cleaned up a bit, the search-and-replace for capacity doesn't read quite right